egs/babel/asr1/conf/lang.conf

```diff
@@ -0,0 +1,292 @@
+# A giant configurations file for all the BABEL languages
+# as well as some training configurations for training HMM-GMM systems
+# for obtaining phoneme level alignments if you really want to do that
+# All paths starting with /export/* are set for the JHU/CLSP grid and shoudl
+# be changed appropriately for other users
+
+# Cantonese
+train_data_dir_101=/export/babel/data/101-cantonese/release-current/conversational/training
+train_data_list_101=./conf/lists/101-cantonese/train.LimitedLP.list
+train_data_dir_101_FLP=/export/babel/data/101-cantonese/release-current/conversational/training
+train_data_list_101_FLP=./conf/lists/101-cantonese/train.FullLP.list
+dev10h_data_dir_101=/export/babel/data/101-cantonese/release-current/conversational/dev
+dev10h_data_list_101=./conf/lists/101-cantonese/dev.list
+lexicon_file_101=/export/babel/data/101-cantonese/release-current/conversational/reference_materials/lexicon.sub-train1.txt
```

Comment:
Are lexicon related configs required?

Reply:
They aren't needed, but I figured it wouldn't hurt to include them since they were already there, and someone might decide to use them maybe in the future.

Reply:
Sorry. Actually, I am currently using it in the recipe to prepare the data/train/text file. I don't use the pronunciations, but I do use the words in order to filter out word fragments.

Reply:
OK. Keep it then.